lib/src/swipe.dart

```diff
@@ -247,7 +247,9 @@ class _SwipeState<Option> extends State<Swipe<Option>>
       return AnimatedBuilder(
         animation: animationController,
         builder: (context, gestureDetector) {
-          final opacity = math.sqrt(animationController.value);
+          final openRatio =
+              math.sqrt(animationController.value / widget.opensToPosition);
+          final opacity = math.min(openRatio, 1.0);
```

Comment:
💅 very tiny nit, can be ignored, but you could also do:
`final opacity = openRatio.clamp(.0, 1.0);`

Reply:
https://github.com/xaynetwork/xayn_swipe_it/pull/5/commits/4959714536ca00ac2766014fc35b02053f30aa08